models/src/main/thrift/content/Platform.thrift

```diff
@@ -0,0 +1,29 @@
+namespace scala com.gu.contentapi.client.model.v1
+
+include "CapiDateTime.thrift"
+
+struct Platforms {
+  0: optional WebPlatform web;
+
+  1: optional PrintPlatform print;
+
+  2: optional EditionPlatform edition;
+}
+
+struct WebPlatform {
+  0: required string url
+
+  1: required string title
+
+  2: required CapiDateTime.CapiDateTime publicationDate
+}
+
+struct PrintPlatform {
+  0: required i32 pageNumber
+    
+  1: required CapiDateTime.CapiDateTime editionDate
```

Comment:
Do we want the publication name too to differentiate `The Guardian` and `The Observer` (and `Auto Trader`)?

Reply:
I didn't think about that. Is it print only?

Reply:
I think so, that's certainly how we model it in Grid. @blishen will likely know for certain.

Reply:
Done by Simon.